webdriver/tests/bidi/script/call_function/remote_values.py

```diff
@@ -39,3 +39,129 @@ async def test_remote_value_promise(bidi_session, top_context, await_promise):
         assert result == {"type": "number", "value": 42}
     else:
         assert result == {"type": "promise"}
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("await_promise", [True, False])
+async def test_window_context_top_level(bidi_session, top_context,
+                                        await_promise):
+    function_declaration = f"() => window"
+    if await_promise:
+        function_declaration = "async" + function_declaration
+
+    result = await bidi_session.script.call_function(
+        function_declaration=function_declaration,
+        await_promise=await_promise,
+        target=ContextTarget(top_context["context"]),
+        serialization_options=SerializationOptions(max_object_depth=1),
+    )
+
+    recursive_compare(
+        {
+            "type": "window",
+            "value": {
+                "context": top_context["context"]
+            }
+        }, result)
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("await_promise", [True, False])
+async def test_window_context_same_id_after_cross_origin_navigation(
+        bidi_session, top_context, await_promise, test_page_cross_origin):
+    function_declaration = f"() => window"
+    if await_promise:
+        function_declaration = "async" + function_declaration
+
+    result = await bidi_session.script.call_function(
+        function_declaration=function_declaration,
+        await_promise=await_promise,
+        target=ContextTarget(top_context["context"]),
+        serialization_options=SerializationOptions(max_object_depth=1),
+    )
+
+    original_context = result['value']['context']
+
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"],
+        url=test_page_cross_origin,
+        wait="complete")
+    
+    result = await bidi_session.script.call_function(
+        function_declaration=function_declaration,
+        await_promise=await_promise,
+        target=ContextTarget(top_context["context"]),
+        serialization_options=SerializationOptions(max_object_depth=1),
+    )
+
+    cross_origin_context = result['value']['context']
+
+    assert original_context == cross_origin_context
```

Comment:
The expected value is usually on the right side.

```suggestion
    assert cross_origin_context == original_context
```